sensor/common/metrics/metrics.go

```diff
@@ -386,9 +395,18 @@ func IncrementTotalProcessesSentCounter(numberOfProcesses int) {
 	totalProcessesSentCounter.Add(float64(numberOfProcesses))
 }
 
-// IncrementTotalProcessesReceivedCounter increments the total number of endpoints received
-func IncrementTotalProcessesReceivedCounter(numberOfProcesses int) {
-	totalProcessesReceivedCounter.Add(float64(numberOfProcesses))
+// IncrementTotalProcessesReceivedCounter increments the total number of processes received
+func IncrementTotalProcessesReceivedCounter() {
+	totalProcessesCounter.With(prometheus.Labels{
+		"Operation": "added",
```

Comment:
nit: we have this [metrics.Add.String()](https://github.com/stackrox/stackrox/blob/6a15c1e89a503cddf0681d4f906e2f9b854b85b6/pkg/metrics/operations.go#L10) constants that we normally use in this situations like for the [bufferedStream](https://github.com/stackrox/stackrox/blob/6a15c1e89a503cddf0681d4f906e2f9b854b85b6/sensor/common/metrics/metrics.go#L463-L476). You can also used them for remove and drop operations.